src/main/java/uk/ac/ebi/ampt2d/metadata/persistence/entities/Study.java

```diff
@@ -38,11 +34,12 @@
 @Entity
 public class Study {
 
-    @ApiModelProperty(position = 1, value = "Study auto generated id", required = true, readOnly = true)
-    @JsonProperty(access = JsonProperty.Access.READ_ONLY)
+    @ApiModelProperty(position = 1, required = true)
+    @Size(min = 1, max = 255)
+    @NotNull
+    @JsonProperty
     @Id
-    @GeneratedValue(strategy = GenerationType.AUTO)
-    private long id;
+    private String accession;
```

Comment:
as we have versions in accessions , should we have accessions as id ?

Reply:
Normally a versioned accession could be represented as "$accession.$version". I think we can use that as the ID. What do you think? @cyenyxe and @jorizci

Reply:
As last days have proved this could be more complex, imho, Id and versions should not be linked. If we have to support versioning or history of the accession then we have to devise a different structure. Or procedure. It can be an independant id and a indexed accession column plus a version column.

I don't recall what is specified on the user stories. @cyenyxe?